src/span_ext.rs

```diff
@@ -278,17 +326,25 @@ impl OpenTelemetrySpanExt for tracing::Span {
             };
             let mut key = Some(key.into());
             let mut value = Some(value.into());
```

Comment:
```suggestion
            let mut key_value = Some(KeyValue::new(key.into(), value.into()));
```

and then use that directly.

Arguably, it should have already been written that way but now that there are two places where `KeyValue` is constructed, let's just extract it.